package.json

```diff
@@ -31,5 +37,15 @@
   },
   "devDependencies": {
     "jest": "^23.6.0"
+  },
+  "bugs": {
+    "url": "https://github.com/bnb/good-first-issue/issues",
+    "email": "hello@bnb.im,dhruvjainpenny@gmail.com"
+  },
+  "contributors": [
+    "Dhruv Jain <dhruvjainpenny@gmail.com> (https://maddhruv.github.io)"
+  ],
+  "engines": {
+    "node": ">=8.0.0"
```

Comment:
Node.js 8.0.0 is not LTS and is not secure. Please change this to `>=8.11.4`, which is the current minimum secure version of the Node.js 8 release line.